lib/krane/kubernetes_resource.rb

```diff
@@ -409,11 +411,26 @@ def self.go_template_for(kind, name)
       def self.extract_all_from_go_template_blob(blob)
         blob.split(EVENT_SEPARATOR).map do |event_blob|
           pieces = event_blob.split(FIELD_SEPARATOR, FIELDS.length)
+
+          series = pieces[FIELDS.index(".series")]
+          last_timestamp = pieces[FIELDS.index(".lastTimestamp")]
+
+          # Find the right count and last_timestamp according to Events API version
+          count, last_timestamp = if series.present? && series != "<no value>" # Events/v1
+            count_regex = /count:(?<value>\S+?(?=\s))/
```

Comment:
Any chance `JSON.parse(series)` works?

Reply:
hmmmm, I didn't knew about it and will try, thanks for the suggestion 👍

Reply:
Unfortunately, it does not work because `series` doesn't contain a valid json string, this is the content of the `series` field:

```ruby
[13] pry(Krane::KubernetesResource::Event)> blob
=> "PodENDFIELD--BEGINFIELDpvcENDFIELD--BEGINFIELD<no value>ENDFIELD--BEGINFIELD<no value>ENDFIELD--BEGINFIELDFailedSchedulingENDFIELD--BEGINFIELD0/1 nodes are available: 1 node(s) didn't find available persistent volumes to bind.ENDFIELD--BEGINFIELD2019-11-08T18:56:08.886672ZENDFIELD--BEGINFIELD<no value>ENDEVENT--BEGINEVENTPodENDFIELD--BEGINFIELDpvcENDFIELD--BEGINFIELD<no value>ENDFIELD--BEGINFIELD<no value>ENDFIELD--BEGINFIELDFailedSchedulingENDFIELD--BEGINFIELD0/1 nodes are available: 1 node(s) didn't find available persistent volumes to bind.ENDFIELD--BEGINFIELD2019-11-08T18:56:08.900067ZENDFIELD--BEGINFIELDmap[count:1 lastObservedTime:2019-11-08T18:56:10.183912Z]ENDEVENT--BEGINEVENT"
[14] pry(Krane::KubernetesResource::Event)> series
=> "map[count:1 lastObservedTime:2019-11-08T18:56:10.183912Z]"
```